dev_setup.sh

```diff
@@ -184,6 +184,12 @@ if ! found_exe android-studio ; then
     fi
 fi
 
+if ! gem list '^bundler$' -i --version 2.1.4; then
```

Comment:
Will this work on ubuntu too?

Reply:
Looks like it might need an executable check on `gem`, and `sudo apt-get install rubygems` if not found

Reply:
For now, we don't need ruby installed on anything but a macOS. I don't think people can develop on ubuntu for apple (since xcode tooling requires macOS to be installed). I can definitely throw a check around this so it doesn't error out for users on ubuntu though.

Reply:
Added an OS check for the install